app/helpers/sufia/sufia_helper_behavior.rb

```diff
@@ -4,6 +4,14 @@ def orcid_label(style_class='')
       "#{image_tag 'orcid.png', { alt: t('sufia.user_profile.orcid.alt'), class: style_class }} #{t('sufia.user_profile.orcid.label')}".html_safe
     end
 
+    def zotero_label(style_class='')
```

Comment:
Making this argument take a hash of options would make it easier to see that the argument is a class. e.g. `zotero_label(html_class: 'orcid')`